schedule/storefixedshifts.go

```diff
@@ -64,6 +73,24 @@ func (store *Store) FixedShiftGroups(ctx context.Context, tx *sql.Tx, scheduleID
 		}
 	}
 
+	check, err := store.usr.UserExists(ctx)
+	if err != nil {
+		return nil, err
+	}
+	defer check.Done()
+
+	for i, tmp := range data.V1.TemporarySchedules {
```

Comment:
nit: comment that we are removing users that no longer exist here for future maintenance